packages/go/analysis/ad/ad.go

```diff
@@ -856,7 +856,8 @@ func GetADCSESC6aEdgeComposition(ctx context.Context, db graph.Database, edge *g
 				return nil
 			}
 		}); err != nil {
-			return false, err
+			closureErr = fmt.Errorf("could not fetch cert template node: %w", err)
+			return false
```

Comment:
This was the only time we actually used the error result properly in practice. Added a closureErr to allow us to hinge on an error here to propagate an error up the stack.